src/features/homepage/Card.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './css/Card.css';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import {
+  SiIbm, SiMicrosoft, SiApple, SiAmazon,
+} from 'react-icons/si';
+import { TbBrandMeta } from 'react-icons/tb';
+
+export const Img = ({ symbol }) => {
+  if (symbol === 'MSFT') {
+    return <SiMicrosoft className="logos" />;
+  } if (symbol === 'IBM') {
+    return <SiIbm className="logos" />;
+  } if (symbol === 'AAPL') {
+    return <SiApple className="logos" />;
+  } if (symbol === 'AMZN') {
+    return <SiAmazon className="logos" />;
+  } if (symbol === 'META') {
+    return <TbBrandMeta className="logos" />;
+  }
+  return <img src="" alt="not found" />;
+};
+
+const Card = (props) => {
+  const { quote } = props;
+  return (
+    <section className="card">
+      <header id="card-header">
+        <div id="imgContainer">
+          <Img symbol={quote.ticker} />
+        </div>
+        <Link to="details" state={quote}><BsArrowRightCircle id="enterIcon" /></Link>
+      </header>
+      <h2>
+        {quote.ticker}
+      </h2>
+      <p id="card-price">
+        {quote.price}
+        $
+      </p>
+    </section>
+  );
+};
+
+export default Card;
+
+Card.propTypes = {
+  quote: PropTypes.shape({
+    ticker: PropTypes.string,
+    price: PropTypes.number,
+  }).isRequired,
+};
+Img.propTypes = {
+  symbol: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] There are errors flagging for prototypes in this component that are causing the app to not run. Kindly fix the errors, test the app if it runs properly, and resubmit again 👍🏽

<img width="1776" alt="Screenshot 2023-02-09 at 12 55 29 AM" src="https://user-images.githubusercontent.com/50941074/217688980-0eeb3e9c-d937-497d-b8b4-b0afbe4c2a31.png">

Reply:
Hello, 
It is working fine from my end. I think the error is caused by not receiving data from the API. Kindly use the API key if you run the code locally.